app/schemas/com.github.dedis.student20_pop.model.data.LAODatabase/1.json

```diff
@@ -0,0 +1,320 @@
+{
+  "formatVersion": 1,
+  "database": {
+    "version": 1,
+    "identityHash": "af8855922b2f6d3d9aeff68c7c58db43",
+    "entities": [
+      {
+        "tableName": "LAO",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`channel` TEXT NOT NULL, `id` TEXT, `name` TEXT, `lastModifiedAt` INTEGER, `createdAt` INTEGER, `organizer` TEXT, `modificationId` TEXT, PRIMARY KEY(`channel`))",
+        "fields": [
+          {
+            "fieldPath": "channel",
+            "columnName": "channel",
+            "affinity": "TEXT",
+            "notNull": true
+          },
+          {
+            "fieldPath": "id",
+            "columnName": "id",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "name",
+            "columnName": "name",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "lastModifiedAt",
+            "columnName": "lastModifiedAt",
+            "affinity": "INTEGER",
+            "notNull": false
+          },
+          {
+            "fieldPath": "createdAt",
+            "columnName": "createdAt",
+            "affinity": "INTEGER",
+            "notNull": false
+          },
+          {
+            "fieldPath": "organizer",
+            "columnName": "organizer",
+            "affinity": "TEXT",
+            "notNull": false
+          },
+          {
+            "fieldPath": "modificationId",
+            "columnName": "modificationId",
+            "affinity": "TEXT",
+            "notNull": false
+          }
+        ],
+        "primaryKey": {
+          "columnNames": [
+            "channel"
+          ],
+          "autoGenerate": false
+        },
+        "indices": [],
+        "foreignKeys": []
+      },
+      {
+        "tableName": "Meeting",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`originalId` TEXT NOT NULL, `id` TEXT, `laoChannel` TEXT, `name` TEXT, `creation` TEXT, `location` TEXT, `start` INTEGER, `end` INTEGER, `extra` TEXT, `modificationId` TEXT, PRIMARY KEY(`originalId`))",
+        "fields": [
+          {
+            "fieldPath": "originalId",
```

Comment:
What is the difference between the `originalId` and `id` field?

Reply:
The meeting id might change because of a modification of the name field since `id` depends on a hash of `name`. The `originalId` refers to the initial id the meeting was created with

Reply:
I don't think it's useful to save the `originalId`, I don't know if it's needed or will be used. 

If it is needed somewhere then I think it would be good to have the same for a LAO, the LAO `id` depends on the `name` too and the name can be modified (when updating a LAO)